packages/excalidraw/components/App.tsx

```diff
@@ -2988,6 +2991,8 @@ class App extends React.Component<AppProps, AppState> {
           ),
         });
 
+        this.actionManager.executeAction(actionFinalize);
```

Comment:
This removes the selection from the freshly pasted image, which leads to test failures. You can run the tests locally to see by `yarn test:app`.

The expected case is that after pasting the pasted image is selected.

Reply:
Sure, that makes sense. I'll implement the changes